models/gc_tiger_lb2/utils/TigerLB2Runner.py

```diff
@@ -0,0 +1,59 @@
+"""
+------------------------------------------------
+Mhub / DIAG - Run Module for Tiger LB2 Algorithm
+------------------------------------------------
+
+------------------------------------------------
+Author: Sil van de Leemput
+Email:  sil.vandeleemput@radboudumc.nl
+------------------------------------------------
+"""
+from mhubio.core import Instance, InstanceData, IO, Module, ValueOutput, Meta, DataType, FileType
+
+from pathlib import Path
+import numpy as np
+import SimpleITK as sitk
+import torch
+
+import sys
+import json
+
+
+@ValueOutput.Name('til_score')
+@ValueOutput.Meta(Meta(key="value"))
+@ValueOutput.Label('TIL score')
+@ValueOutput.Type(int)
+@ValueOutput.Description('percentage of stromal area covered by tumour infiltrating lymphocytes. Values between 0 (percent) to 100 (percent).')
+class TilScoreOutput(ValueOutput):
+    pass
+
+
+class TigerLB2Runner(Module):
+
+    CLI_SCRIPT_PATH = Path(__file__).parent.parent / "scripts" / "tiger_lb2_cli.py"
+
+    @IO.Instance()
+    @IO.Input('in_data', 'tiff:mod=sm', the='input whole slide image Tiff')
+    @IO.Output('out_data', 'gc_tiger_lb2_til_score.json', 'json:model=TigerLB2TILScore', 'in_data', the='TIGER LB2 TIL score')
+    @IO.OutputData('til_score', TilScoreOutput, data='in_data', the='TIGER LB2 TIL score - percentage of stromal area covered by tumour infiltrating lymphocytes. Values between 0-100 (percent).')
+    def task(self, instance: Instance, in_data: InstanceData, out_data: InstanceData, til_score: TilScoreOutput) -> None:
+        if not torch.cuda.is_available():
+            raise NotImplementedError("TigerLB2Runner requires CUDA to be available!")
+
+        # Execute the Tiger LB2 Algorithm through a Python subprocess
+        self.subprocess(
+            [
+                sys.executable,
+                str(self.CLI_SCRIPT_PATH),
+                in_data.abspath,
+                out_data.abspath,
+            ]
+        )
+
+        if not Path(out_data.abspath).is_file():
```

Comment:
This is not required; our `IO.Output` decorator performs a file-existence check and confirms only existing files using the official data API (`data.confirm()`).

Reply:
If required for a subsequent step, pls invert the expression instead of throwing exceptions/errors.

Reply:
I think doing it this way is preferable since the raised error clearly communicates what the issue is within the module. If the expression is inverted it would become a silent failure, which is much harder to debug.

Reply:
`Something went wrong` does not at all `clearly communicate what the issue` is.  
Either improve or update as suggested.